src/pipecat/services/cartesia/tts.py

```diff
@@ -166,7 +166,7 @@ def _build_msg(
             "output_format": self._settings["output_format"],
             "language": self._settings["language"],
             "add_timestamps": add_timestamps,
-            "use_original_timestamps": True,
+            "use_original_timestamps": True if self.model_name == "sonic-2" else False,
```

Comment:
`use_original_timestamps: True` only fails for `sonic` (and works for `sonic-2` and `sonic-turbo`. Let's assume future models will continue to support this and only opt out original sonic models.

```suggestion
            "use_original_timestamps": False if self.model_name == "sonic" else True,
```

Reply:
I’ll change it.